### PR TITLE
✨ feat(packages) Add Remix GraphQL scaffold

### PR DESCRIPTION
**Pull request by enZane**
# What this PR does?
- Updates the esbuild config to support copy the static files for the templates
- Add a dev command that runs esbuild when a change happens in src folder
- Added the scaffolding files required for remix-graphql

### CODE REVIEW

File: packages/create-drupal-decoupled/src/templates/remix-graphql/env.example [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DRUPAL_AUTH_URI=
+DRUPAL_CLIENT_ID=
+DRUPAL_CLIENT_SECRET=
+DRUPAL_GRAPHQL_URI=
```
Comment (bitrvmpd) [inline packages/create-drupal-decoupled/src/templates/remix-graphql/env.example:4]: New line is missing here

File: packages/create-drupal-decoupled/src/helpers/scaffolding.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  remix: {
+    files: [
+      {
+        folderPath: 'app/utils',
+        fileName: 'drupal-client.server.ts',
+      },
+      {
+        folderPath: '.',
+        fileName: 'env.example',
+        // This is due the copy plugin from esbuild, it doesn't allow to copy files with a dot at the beginning
```
Comment (bitrvmpd) [inline packages/create-drupal-decoupled/src/helpers/scaffolding.ts:15]: If we do a `cp -r` this won't be an issue

**Review by bitrvmpd — CHANGES_REQUESTED** [review]

File: packages/create-drupal-decoupled/src/helpers/scaffolding.ts (hunk 2 above)
Comment (enZane) [reply packages/create-drupal-decoupled/src/helpers/scaffolding.ts:15]: You right

**Review by bitrvmpd — APPROVED** [review]
LGTM

**Pull request merged**